### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by TeoDoe777** [asmcss/assembler#29] [Q&A]
Good day, I have read the documentation several times. But I can not understand this

We have a structure
`<div class="class1"><div class="class2"><div class="class3">hello world</div></div></div>`

how can I use your framework to change class2 on hovering over class1 e.g. the same "color"

**Comment by sorinsarca** [comment]
Sorry, but I don't understand your question.

**Reply by TeoDoe777** [reply]
I forgot to throw in the css code 
I immediately added !important because I also could not find how to work with it through the framework


```css
.class1:hover .class3 {
color:tomato !important;
}
```
